test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -18,14 +18,6 @@ package benchmark
 
 // High Level Configuration for all predicates and priorities.
 type schedulerPerfConfig struct {
-	NodeCount    int // The number of nodes which will be seeded with metadata to match predicates and have non-trivial priority rankings.
-	PodCount     int // The number of pods which will be seeded with metadata to match predicates and have non-trivial priority rankings.
-	NodeAffinity *nodeAffinity
-	// TODO: Other predicates and priorities to be added here.
-}
-
-// nodeAffinity priority configuration details.
-type nodeAffinity struct {
-	nodeAffinityKey string // Node Selection Key.
-	LabelCount      int    // number of labels to be added to each node or pod.
```

Comment:
The original design wanted to have an explicit control over the seeded meta-data, for example how many labels do we want to have will be helpful.
ref: https://github.com/kubernetes/kubernetes/issues/45973#issue-229422318.